bigquery/snippets/snippet.go

```diff
@@ -993,7 +996,7 @@ func runAndRead(ctx context.Context, client *bigquery.Client, q *bigquery.Query)
 	}
 	// [END bigquery_query]
 	// [END bigquery_query_destination_table]
-	// [END bigquery_query_legacy]
```

Comment:
I think `bigquery_query_legacy` needs to stay. Add a new start/end pair for `bigquery_query_legacy_large_results`